packages/enketo-express/Dockerfile

```diff
@@ -1,18 +1,16 @@
-FROM node:16
+FROM node:20
 
 ENV ENKETO_SRC_DIR=/srv/src/enketo_express
 WORKDIR ${ENKETO_SRC_DIR}
 
-RUN npm install -g pm2@$(npm info pm2 version)
-
 COPY . ${ENKETO_SRC_DIR}
 
-RUN npm install -g npm@^6 && npm clean-install && npx grunt && npm prune --production
+RUN yarn install --frozen-lockfile && yarn grunt && yarn install --production
 
 # Persist the `secrets` directory so the encryption key remains consistent.
```

Comment:
This is no longer needed -- we're not doing secrets management at all. Users are responsible for putting their own secrets in the config file.